src/QuokkaSimulation.hpp

```diff
@@ -1407,20 +1545,21 @@ auto QuokkaSimulation<problem_t>::advanceHydroAtLevel(amrex::MultiFab &state_old
 				const amrex::IntVect cell_idx = redoFlag.maxIndex(0);
 				// Calculate the coordinates based on the cell index and cell size
 				printCoordinates(lev, cell_idx);
-				amrex::print_state(stateNew, cell_idx);
+				amrex::print_state(stateNew_cc, cell_idx);
 			}
 
 			// synchronize redoFlag across ranks
 			redoFlag.FillBoundary(geom[lev].periodicity());
 
-			// replace fluxes around troubled cells with Godunov fluxes
-			replaceFluxes(fluxArrays, FOfluxArrays, redoFlag);
-			replaceFluxes(faceVel, FOfaceVel, redoFlag); // needed for dual energy
+			// TODO(Neco or someone else)
+			// // replace fluxes around troubled cells with Godunov fluxes
+			// replaceFluxes(fluxArrays, FOfluxArrays, redoFlag);
+			// replaceFluxes(faceVel, FOfaceVel, redoFlag); // needed for dual energy
```

Comment:
This is why FOFC doesn't work. None of the high-order fluxes are actually replaced with their first-order counterparts. The emfs also need to be replaced here.